src/libpgmoneta/json.c

```diff
@@ -738,6 +756,25 @@ fill_value(char* str, char* key, uint64_t* index, struct json* o)
       idx++;
       while (idx < len && str[idx] != '"')
       {
+         // check for escape characters in value
+         if (str[idx] == '\\')
+         {
+            idx++;
+            if (idx == len) // security check
+            {
+               goto error;
+            }
+            switch (str[idx])
+            {
+               case '\'': // single quote
+               case '\"': // double quote
+                  val = pgmoneta_append_char(val, str[idx++]);
+                  break;
+               default:
+                  goto error;
+            }
+            continue;
+         }
```

Comment:
Looks good~ Perhaps wrap the code as a static handle_escape_char() to avoid duplicate code.